src/Kunstmaan/MediaBundle/Resources/config/services.yml

```diff
@@ -125,3 +125,40 @@ services:
         calls:
             - [setMimeTypeGuesser, ['@kunstmaan_media.mimetype_guesser.factory']]
             - [setExtensionGuesser, ['@kunstmaan_media.extension_guesser.factory']]
+
+    kunstmaan_media.imagine.filter.loader.background:
+        class: '%liip_imagine.filter.loader.background.class%'
+        arguments:
+            - '@liip_imagine'
+        tags:
+            - { name: 'liip_imagine.filter.loader', loader: 'background' }
+
+    kunstmaan_media.imagine.controller.imagine:
+        class: Kunstmaan\MediaBundle\Helper\Imagine\ImagineController
+        arguments:
+            - '@liip_imagine.service.filter'
+            - '@liip_imagine.data.manager'
+            - '@liip_imagine.cache.signer'
+        tags:
+            - { name: 'controller.service_arguments' }
+        public: true
+
+    kunstmaan_media.imagine.resolver.web_path:
```

Comment:
Do we need still the service id's? Can't we just use the classname as service id?

Reply:
Good point, I will use fqcn's for the new services!

Reply:
@sandergo90 Fixed! Only the `kunstmaan_media.imagine.filter.loader.background` service could not renamed as the class is based on a parameter. This could be updated in v6